src/core_plugins/kibana/public/discover/index.html

```diff
@@ -90,20 +90,20 @@
 
                   &mdash;
 
-                  <span class="results-interval" ng-hide="showInterval">
-                    <a
-                      ng-click="toggleInterval()">
-                      {{ intervalName }}
-                    </a>
-                  </span>
-
-                  <span ng-show="showInterval" class="results-interval form-inline">
+                  <span class="results-interval form-inline">
                     <select
                       class="form-control"
                       ng-model="state.interval"
                       ng-options="interval.val as interval.display for interval in intervalOptions | filter: intervalEnabled"
+                      ng-blur="toggleInterval()"
                       >
                     </select>
+                    <kbn-info
+                      ng-show="bucketInterval.scaled"
+                      placement="right"
+                      class="text-warning"
```

Comment:
Can you change this class to `kuiIcon--info`? It's a bit of a misuse of the class but I think it's OK in this instance because I doubt that class will change.